### ISSUE TITLE
ctypes mishandles `void` return type

### ISSUE THREAD

**Issue by 6a48b898-0867-4d13-9fe9-1317cbf134b1** [python/cpython#91353]
BPO | [47197](https://bugs.python.org/issue47197)
--- | :---
Nosy | @amauryfa, @abalkin, @tiran, @meadori, @hoodmane, @hoodmane
PRs | <li>python/cpython#32246</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2022-04-02.01:14:09.315>
labels = ['ctypes', 'type-bug', 'tests', '3.11']
title = 'ctypes mishandles `void` return type'
updated_at = <Date 2022-04-03.21:02:00.133>
user = 'https://github.com/hoodmane'
```

bugs.python.org fields:
```python
activity = <Date 2022-04-03.21:02:00.133>
actor = 'christian.heimes'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Tests', 'ctypes']
creation = <Date 2022-04-02.01:14:09.315>
creator = 'hoodchatham'
dependencies = []
files = []
hgrepos = []
issue_num = 47197
keywords = ['patch']
message_count = 1.0
messages = ['416526']
nosy_count = 6.0
nosy_names = ['amaury.forgeotdarc', 'belopolsky', 'christian.heimes', 'meador.inge', 'hoodmane', 'hoodchatham']
pr_nums = ['32246']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue47197'
versions = ['Python 3.11']
```

</p></details>

**Comment by corona10** [comment]
I think that we can close this issue. Please feel free to reopen this issue if needed.

**Comment by tiran** [comment]
Yeah, the issue is fixed in 3.11 and main branches.
